cmd/sqlflowserver/main_test.go

```diff
@@ -158,9 +189,16 @@ func TestEnd2EndMySQL(t *testing.T) {
 	}
 	dbStr := "mysql://root:root@tcp/?maxAllowedPacket=0"
 	modelDir := ""
-	go start(dbStr, modelDir)
+	caDir, _ := ioutil.TempDir("/tmp", "sqlflow_ssl_")
+	caCrt, caKey, err := generateCA(caDir)
+	defer os.RemoveAll(caDir)
```

Comment:
Given the fact that we always call `ioutil.TempDir` and `generateCA` together, how about we merge them into one:

```go
// generateTempCA creates a pair of CA certificate and key in a temporary directory. It is the caller's responsibility to remove this directory.
func generateTempCA() (tmpDir, caCrt, caKey string, err error) {
```

So we can call it as

```go
tmpDir, caCrt, caKey, err := generateTempCA()
defer os.RemoveAll(tmpDir)
```

Reply:
Done.